### PR TITLE
Add environment configuration to ApplicationConfig

### PR DESCRIPTION
**Pull request by floriangeyer21**

### CODE REVIEW

File: src/main/java/com/spring/test/config/ApplicationConfig.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @Bean
+    public LocalSessionFactoryBean getSessionFactory() {
+        LocalSessionFactoryBean factoryBean = new LocalSessionFactoryBean();
+        factoryBean.setDataSource(getDataSource());
+        Properties properties = new Properties();
+        properties.put("hibernate.format_sql", "true");
+        properties.put("hibernate.generate_statistics", "true");
+        properties.put("hibernate.hbm2ddl.auto", "create-drop");
```
Comment (margarita-ignatieva) [inline src/main/java/com/spring/test/config/ApplicationConfig.java]: properties for hibernate are also received from db.properties

**Review by margarita-ignatieva — APPROVED** [review]

File: src/main/java/com/spring/test/config/ApplicationConfig.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+
+@Configuration
+@PropertySource("classpath:application.properties")
+@ComponentScan(basePackages = {
+        "com.spring.test.repository",
+        "com.spring.test.service"
+})
+public class ApplicationConfig {
+    private final Environment env;
```
Comment (RomanDubovskyi) [inline src/main/java/com/spring/test/config/ApplicationConfig.java]:
```suggestion
    private final Environment environment;
```

**Review by RomanDubovskyi — COMMENTED** [review]
change PR name it's uninformative 
remove files from previous PR

**Comment by RomanDubovskyi** [comment]
PR name wasn't changed 
image

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**